src/main/resources/config/application.yml

```diff
@@ -219,6 +219,9 @@ info:
     # default value set to false
     text-assessment-analytics-enabled: false
     browser-fingerprints-enabled: true
+    # Disables the password-based login user interface, but leaves the API enabled.
+    # Can be useful with alternative login flows (e.g., SAML2).
+    password-login-disabled: false
```

Comment:
can we move this into a different section? I think it should be part of the section where the SAML2 related information is stored

Reply:
Sure, I will move it to the SAML2 config (`info.saml2.password-login-disabled`).